### DISCUSSION TITLE
Is there any documentation regarding preloading applications?

### DISCUSSION THREAD

**Discussion by RafayAK** [nestybox/sysbox#375] [General] [answered]
Hi, I've gone through the sysbox documentation, and successfully set up my application in a sysbox systemd container, while in the container's interactive shell mode. 

I would like to know if there is any documentation regarding preloading applications in a sysbox container? Can I just build on top of this container and perform normal docker `RUN` commands to install packages and applications from `apt` and have them run on container start as they create systemd unit files?

**Comment by ctalledo** [comment, marked as answer]
Hi @RafayAK ,

Thanks for giving Sysbox a shot, glad you got it working.

> Can I just build on top of this container and perform normal docker RUN commands to install packages and applications from apt and have them run on container start as they create systemd unit files?

Yes exactly, that's the way you would do it. The Dockerfiles for the sample container images we provide are [here](https://github.com/nestybox/dockerfiles/tree/master). Feel free to copy them and adapt them to your needs. 

In addition, if you plan to use Docker inside the Sysbox container, you can also preload the inner container images into the container (so that when you launch the sysbox container, those inner container images are already in it and don't need to be downloaded from the network). To do that, see [this doc](https://github.com/nestybox/sysbox/blob/master/docs/quickstart/images.md).

Hope this helps!

**Reply by RafayAK** [reply]
Perfect! Thanks!
